application/gqlgen/user.go

```diff
@@ -202,7 +209,7 @@ func (r *mutationResolver) UpdateUser(ctx context.Context, input UpdateUserInput
 		}
 	}
 
-	if err := user.Save(); err != nil {
+	if err = user.Save(); err != nil {
```

Comment:
I don't think it hurt that you removed the colon, but I also don't think it was necessary (because of the `if`).

Reply:
Just silencing a lint warning.